blog/models.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.conf import settings
+
+
+# Create your models here.
+class User(AbstractUser):
+    pass
```

Comment:
The User model is currently empty. If you're planning to add custom fields to it in the future, this is fine. But if not, you could directly use Django's built-in User model instead of creating a custom one.